### PR TITLE
once more, as a joke - re-ports the proto-kinetic glaive

### PR DESCRIPTION
**Pull request by Hatterhat**
## About The Pull Request
adds the proto-kinetic glaive as an alt reskin. if i come up with a good way to differentiate it it'll be a new item Eventually

original concept Citadel-Station-13/Citadel-Station-13#13003 (there i am, gary!)
resprite from Citadel-Station-13/Citadel-Station-13#15773, and then further modified by me when i made it 64x64

## How This Contributes To The Skyrat Roleplay Experience
it doesn't but hey at least it looks neat
## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>

https://github.com/Skyrat-SS13/Skyrat-tg/assets/31829017/8b3084c3-ec6a-4b98-8961-475ae6bbec91

https://github.com/Skyrat-SS13/Skyrat-tg/assets/31829017/dd5930de-636b-4e40-af86-fd46c9005b46

https://github.com/Skyrat-SS13/Skyrat-tg/assets/31829017/82093edb-3260-4f86-96f1-d013d60692a5

</details>

## Changelog

:cl:
add: Proto-kinetic crushers can now be reskinned via alt-click; the only other skin is the proto-kinetic glaive. It has a neat desc and inhand.
/:cl:

### CODE REVIEW

**Comment by kittysmooch** [comment]
can it pretty please be a roundstart alt or reskin of the crusher 👉 👈

**Comment by SpaceVampire** [comment]
If it is going to be exactly the same as a crusher in function, having work as a reskin option seems like it would make more sense.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
I'm cool with a reskin, but I'd rather have it be using the actual reskin system that we already have in-game, rather than something obscure like hitting a vendor with it.

**Comment by Iamgoofball** [comment]
👎 
needs to include an option to backflip into chasms during megafauna fights

File: modular_skyrat/modules/mining_vendor_additions/code/order_mining.dm [hunk 1]
```diff
@@ -6,47 +6,34 @@
...
-	inhand_x_dimension = 64
-	inhand_y_dimension = 64
-	lefthand_file = 'modular_skyrat/master_files/icons/mob/64x64_lefthand.dmi'
-	righthand_file = 'modular_skyrat/master_files/icons/mob/64x64_righthand.dmi'
-	icon_state = "crusher-glaive"
+/obj/item/kinetic_crusher/post_reskin(mob/our_mob)
+	if(icon_state == "crusher-glaive")
+		name = "proto-kinetic glaive"
+		desc = "A modified proto-kinetic crusher, it is still little more than various mining tools cobbled together \
+		into a high-tech knife on a stick with a handguard and goliath-leather grip. While equally as effective as its unmodified compatriots, \
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/mining_vendor_additions/code/order_mining.dm]:
```suggestion
			into a high-tech knife on a stick with a handguard and goliath-leather grip. While equally as effective as its unmodified compatriots, \
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Only change I have left.

**Review by GoldenAlpharex — APPROVED** [review]
Should be good to go!

**Pull request merged**